v21.2/known-limitations.md

```diff
@@ -159,51 +163,6 @@ UNION ALL SELECT * FROM t1 LEFT JOIN t2 ON st_contains(t1.geom, t2.geom) AND t2.
 
 [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/59649)
 
-## Unresolved limitations
-
-### `IMPORT` into a `REGIONAL BY ROW` table
```

Comment:
@ajstorm, please confirm that it's ok to remove this limitation from the 21.2 docs.

Reply:
Confirmed.  #70270.

Reply:
Thanks, Adam.